data-transformers/transformers/fabanTransformer.py

```diff
@@ -0,0 +1,285 @@
+            query["passed"] = operation.find("passed").text
+            queries.append(query)
+    return queries
+
+def createDriverResponseTimesQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        for operation in driver.find("responseTimes"):
+            for stat in operation:
+                query = {}
+                query["trial_id"] = trialID
+                query["experiment_id"] = experimentID
+                query["host"] = host
+                query["driver_name"] = driver.attrib["name"]
+                query["op_name"] = operation.attrib["name"]
+                if not "passed" in stat.tag:
+                    query["stat_name"] = stat.tag
+                    query["stat_value"] = stat.text
+                else:
+                    continue
+                query["passed"] = operation.find("passed").text
+                queries.append(query)
+    return queries
+
+def createDriverDelayTimesQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        for operation in driver.find("delayTimes"):
+            query = {}
+            query["trial_id"] = trialID
+            query["experiment_id"] = experimentID
+            query["host"] = host
+            query["driver_name"] = driver.attrib["name"]
+            query["op_name"] = operation.attrib["name"]
+            query["type"] = operation.attrib["type"]
+            query["targeted_avg"] = operation.find("targetedAvg").text
+            query["actual_avg"] = operation.find("actualAvg").text
+            query["min"] = operation.find("min").text
+            query["max"] = operation.find("max").text
+            query["passed"] = operation.find("passed").text
+            queries.append(query)
+    return queries
+
+def createDriverCustomStatsQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        for stat in driver.find("customStats"):
+            query = {}
+            query["trial_id"] = trialID
+            query["experiment_id"] = experimentID
+            query["host"] = host
+            query["stat_name"] = driver.find("customStats").attrib["name"]
+            query["driver_name"] = driver.attrib["name"]
+            query["description"] = stat.find("description").text
+            query["result"] = stat.find("result").text
+            queries.append(query)
+    return queries
+
+def createDetailsQuery(data, trialID, experimentID, host):
+    firstSectionIndex = 0
+    for i in range(len(data)):
+        if "Section" in data[i] and "Benchmark" not in data[i]:
+            firstSectionIndex = i
+            break
+    data = data[firstSectionIndex:]
+    
+    queries = []
+    currentOps = []
+    currentSection = ""
+    for line in data:
+        if "Section" in line:
+            currentSection = line.split(":")[1]
+        elif "Time" in line:
```

Comment:
Where do you store the unit of measure of the timing, that is reported next to the word `Time` in the first column?